pkg/api/l2geth.go

```diff
@@ -7,32 +7,30 @@ import (
 	"github.com/tokamak-network/tokamak-titond-backend/pkg/model"
 )
 
-type l2gethConfig struct {
-	data map[string]string
-}
-
-func (t *TitondAPI) CreateL2Geth(data *model.Component /*TODO : get params for config, namespace*/) *model.Component {
-	// TODO : deal with DB
-	// t.db.CreateComponent()
-
-	/*
-		This is currently hardcoding, but
-		will be replaced by taking a value as a parameter and generating it
-	*/
-	namespace := "default"
-	config := &l2gethConfig{
-		data: map[string]string{},
+func (t *TitondAPI) CreateL2Geth(l2geth *model.Component, config *ComponentConfig) (*model.Component, error) {
+	network, err := t.db.ReadNetwork(l2geth.NetworkID)
+	if err != nil {
+		return nil, err
 	}
 
-	config.data["ETH1_CONFIRMATION_DEPTH"] = "1"
-	config.data["GASPRICE"] = "100"
+	config.Namespace = generateNamespace(l2geth.NetworkID)
+	config.Data["ROLLUP_STATE_DUMP_PATH"] = network.StateDumpURL
+	config.Data["ROLLUP_CLIENT_HTTP"] = network.ContractAddressURL
```

Comment:
As you know, we have to set this to `datat-transport-layer`! I think it can be changed after adding create data-transport-layer!

Reply:
oh this is my mistake! thank you! i fixed it!